consensus/istanbul/backend/announce.go

```diff
@@ -497,32 +523,66 @@ func (sb *Backend) handleAnnounceMsg(peer consensus.Peer, payload []byte) error
 		logger.Trace("Going to process an announce msg", "announce records", announceData.AnnounceRecords)
```

Comment:
On line 519, consider continuing instead of bailing out here if the function returned an error.